<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>491</width>
      <height>501</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>割り算</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>桁数</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>割られる数の桁</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="div_num_a">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>割る数の桁</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="div_num_b">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>231</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>あまり</string>
      </property>
      <widget class="QRadioButton" name="div_res_no">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>181</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>なし</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="div_res_yes">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>あり</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="div_res_mix">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>181</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>ミックス</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>191</width>
         <height>121</height>
        </rect>
       </property>
       <property name="title">
        <string>あまり「あり」の数</string>
       </property>
       <widget class="QSlider" name="horizontalSlider">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9</number>
        </property>
        <property name="value">
         <number>4</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSpinBox" name="div_num_v">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9</number>
        </property>
        <property name="value">
         <number>4</number>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>350</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>問題数</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="div_num_l">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>350</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
      <property name="value">
       <number>9</number>
      </property>
     </widget>
     <widget class="QPushButton" name="div_generate">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>400</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>問題作成</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>掛け算</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>350</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>問題数</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mul_generate">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>400</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>問題作成</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="mul_num_l">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>350</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
      <property name="value">
       <number>9</number>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>231</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>小数点</string>
      </property>
      <widget class="QRadioButton" name="mul_no">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>181</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>なし</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="mul_yes">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>あり</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>191</width>
         <height>201</height>
        </rect>
       </property>
       <property name="title">
        <string>小数点以下桁数</string>
       </property>
       <widget class="QSpinBox" name="mul_num_c">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9</number>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="mul_num_d">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9</number>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>ひとつめの数</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>ふたつめの数</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>桁数</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>割られる数の桁</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="mul_num_a">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>割る数の桁</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="mul_num_b">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
